compiler/core/lam_compile.ml

```diff
@@ -49,7 +50,14 @@ let rec apply_with_arity_aux (fn : J.expression) (arity : int list)
       if len >= x then
         let first_part, continue = Ext_list.split_at args x in
         apply_with_arity_aux
-          (E.call ~info:{arity = Full; call_info = Call_ml} fn first_part)
+          (E.call
+             ~info:
+               {
+                 arity = Full;
+                 call_info = Call_ml;
+                 (* no clue if this is correct *) call_transformed_jsx = false;
```

Comment:
maybe use one of the existing default constants for this -- seems likely to be correct as these are generated applications, not coming from jsx ppx